version.go

```diff
@@ -4,6 +4,6 @@ package ipfs
 var CurrentCommit string
 
 // CurrentVersionNumber is the current application's version literal
-const CurrentVersionNumber = "0.1.0"
+const CurrentVersionNumber = "0.2.0"
 
-const ApiVersion = "/go-ipfs/" + CurrentVersionNumber + "/"
+const ApiVersion = "/go-btfs/" + CurrentVersionNumber + "/"
```

Comment:
There's unit test for this,
core/corehttp/option_test.go:23:		return fmt.Sprintf("%s (%s != %s)\n", errAPIVersionMismatch, version.ApiVersion, tc.userAgent)
core/corehttp/option_test.go:33:		{version.ApiVersion, APIPath + "/test", true, "check!", http.StatusOK},
Please check if unit test can pass after your change.

Reply:
i checked locally and it passed.  it also passed in the linked jenkins unit test job which gates merging.